### PR TITLE
Upstream changes

### PR DESCRIPTION
**Pull request by Idclip**

### CODE REVIEW

**Comment by Idclip** [comment]
@richhones

File: CHANGES.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Version 1.0.0 - In Development
     - Added full AX type support for matching VDB Point/Volume types which may
       or may not be registered. Registration of these types still depends on
       on downstream software.
+    - Introduce new single header singles which expose new ax::run() pipeline
```
Comment (richhones) [inline CHANGES.md]: think there is a typo here?

File: openvdb_ax/compiler/PointExecutable.h [hunk 2]
```diff
@@ -47,72 +47,121 @@
...
+    void execute(points::PointDataGrid& grid) const;
+
+    ////////////////////////////////////////////////////////
+
+    /// @brief  Set a specific point group to execute over. The default is none,
+    ///   which corresponds to all points. Note that this can also be compiled
+    ///   into the AX function using the ingroup("mygroup") method.
+    /// @warning  If the group does not exist during execute, a runtime error
+    ///   will be thrown.
+    void setGroupExecution(const std::string&);
```
Comment (richhones) [inline openvdb_ax/compiler/PointExecutable.h]: should these have named args? below in setCreateMissing too

Comment (richhones) [reply openvdb_ax/compiler/PointExecutable.h]: if you are doing this the VolumeExecutable has the same

Comment (richhones) [reply openvdb_ax/compiler/PointExecutable.h]: this could be misread as the arg having no use? (tbh i didnt even realise this would compile)

**Review by richhones — CHANGES_REQUESTED** [review]
Minor cleanup changes requested

File: CHANGES.md (hunk 1 above)
Comment (Idclip) [reply CHANGES.md]: fixed

File: openvdb_ax/compiler/PointExecutable.h (hunk 2 above)
Comment (Idclip) [reply openvdb_ax/compiler/PointExecutable.h]: Added arg names with doxygen docs

**Review by richhones — APPROVED** [review]

**Pull request merged**